### ISSUE TITLE
Add Geom

### ISSUE THREAD

**Issue by ThanosPoulakidas** [google/fonts#4525]
**Font Project Git Repo URL:**
https://github.com/ThanosPoulakidas/geom-font.git

**Super short description of the Font Family:**
Geom, is a contemporary geometric sans serif typeface. It has 7 weights. This typeface is intended for display purposes but it works just as well in small texts. Even though the geometric typefaces look more dynamic in relation with other typefaces, Geom has gone one step further by having an even more additional dynamic characteristics on some of its characters. The typeface is a variable font which includes a Latin and Greek character set.

**Requirements:**

I understand that Google Fonts will publish only fonts that matches its requirements, and I can confirm the project meets them (by ticking the cases, or putting x between the square brackets in text mode):

- [ ] The entire font project is licensed under the [OFL](https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL)
- [ ] There is no "Reserved Font Name" mention in the OFL license
- [ ] Family name is unique according to namecheck.fontdata.com
- [ ] The font supports at least [GF Latin Core glyphset](https://github.com/googlefonts/glyphsets/blob/main/Lib/glyphsets/encodings/GF%20Glyph%20Sets/GF-latin-core_unique-glyphs.nam)
- [ ] The repository respect the [upstream repo structure](https://github.com/googlefonts/gf-docs/tree/main/Spec#upstream-repo-structure)
- [ ] The source files are available in the repository
- [ ] I read, agree and comply with the full [contributing requirements](https://github.com/google/fonts/blob/main/CONTRIBUTING.md)
- [ ] I am the author of the font or the author agreed to this submission
- [ ] I will maintain the repository

**Comment by RosaWagner** [comment]
please @ThanosPoulakidas ping us for review when you have added font binaries to the repo + update readme.md with description and images adapted to your project :)

**Comment by ThanosPoulakidas** [comment]
Thank you @RosaWagner for the answer!
The readme.md has been updated, but as for the font binaries I would like to ask what exactly are those files.
Are you actually reffering to font file formats such as ttf?

Thanks again for your time! :)

**Comment by davelab6** [comment]
@ThanosPoulakidas yes exactly, we'd like you to move the subfolders in `sources` to be within a new top level folder, `fonts` 

Also it seems your Github Actions aren't yet running ( https://github.com/ThanosPoulakidas/Geom/actions/runs/2315937559)

**Comment by ThanosPoulakidas** [comment]
Hi @davelab6. The font files have already been added. However, whenever I create a folder named fonts and move the files inside it in the sources folder, these files seem to disappear. The github app doesn't seem to find them, like they are invinsible or something.

**Comment by ThanosPoulakidas** [comment]
I removed it from .gitignore and now the file has been uploaded successfully.
Still the problem with build font step remains.

**Comment by ThanosPoulakidas** [comment]
@RosaWagner Do you know what is wrong with the build action?
The message's error is: "fontmake.errors.FontmakeError: In 'Geom-Light.glyphs': Loading Glyphs file failed: Junk after plist at line 34869".
But I can't seem to understand what exactly is the error.
Any tip that could help, I would appreciate it!

**Comment by RosaWagner** [comment]
@ThanosPoulakidas we took your submission into account, and we will take a look when we will have time to do so :)  meanwhile, you can look at other repo such as Texturina, Radio Canada, JetBrains etc. or ask for help to [Fontmake](https://github.com/googlefonts/fontmake) if there is an issue with fontmake.

**Comment by ThanosPoulakidas** [comment]
Thank you for your answer! I will look the things you mentioned!

**Comment by ThanosPoulakidas** [comment]
@RosaWagner 
Thank you for your answer!
I have made changes and uploaded the new files.
As for the GF Glyphset for Greek, the polytonic system is not something that fits well with the idea of this typeface. In addition to that there are other fonts that are uploaded withouth having the polytonic greek system or any other system except for the basic system. If you think that by adding them, I will increase the chances or the need of having, I would gladly do it.
As for all the variable part of the font, I am still reading on that part.
Nevertheless, I uploaded the new files for the time being.

**Comment by thlinard** [comment]
> As for the GF Glyphset for Greek, the polytonic system is not something that fits well with the idea of this typeface.

@ThanosPoulakidas Hi, you read the specs too hastily: GF Glyphset for Greek is also about monotonic Greek.

**Comment by ThanosPoulakidas** [comment]
@thlinard You were right.
I uploaded the new files with the right GF Glyphset for Greek for monotonic Greek.

**Comment by ThanosPoulakidas** [comment]
@thlinard You were right.
I uploaded the new files with the right GF Glyphset for Greek for monotonic Greek.
